zadanie2/src/main/java/data.java

```diff
@@ -0,0 +1,73 @@
+import java.text.SimpleDateFormat;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.util.Scanner;
+
+/**
+ * Program sprawdza poprawnosc daty podanej przez uzytkownika.
+ * Daty spoza przedzialu <2001-2099> sa uznawane za niepoprawne.
+ *
+ * @author Kamil Best
+ */
+class Data {
+    final String DATE_FORMAT = "dd-MM-yyyy";
+    private String data;
+    private String rok;
+    private int rokLiczba;
+
+    /**
+     * Metoda pobierająca date od uzytkownika (data w formacie dd-mm-rrrr )
+     * zmienna rok przechowuje 4 koncowe znaki typu char odpowiedzialne za rok, w zmiennej rokLiczba znaki zostaja zamienione na liczby
+     */
+    private void pobierzDate() {
+        System.out.println("Podaj date do sprawdzenia, prawidlowy format: dd-mm-rrrr -> (dzien-miesiac-rok):");
+        System.out.println("Daty spoza okresu (2001-2099) sa niepoprawne:");
+        Scanner d = new Scanner(System.in);
+        data = d.nextLine();
+        rok = data.substring(6, 10);
+        rokLiczba = Integer.parseInt(rok);
+
+    }
+
+    /**
+     * Metoda sprawdza czy rok nalezy do przedzialu <2001,2099>
+     *
+     * @return boolean ( true jezeli nalezy, false jezeli nie nalezy)
+     */
+    private boolean czyPoprawnyRok() {
+        if (rokLiczba < 2001 || rokLiczba > 2099)
+            return false;
+        return true;
+    }
+
+    /**
+     * Metoda sprawdzajaca prawidlowosc daty
+     *
+     * @return boolean (true jezeli prawidlowa, false jezeli nieprawidlowa)
+     * @throws jezeli dane nieprawidlowe
+     */
+    private boolean czyPrawidlowaData() {
```

Comment:
ta metoda jest troche bez sensu, lapiesz wyjatek i zwracasz boolean i nigzie go nie uzywasz.
Proponuje dodac @throws do docblocka, wyjatek lapac wyzej i dac system.out.println wyzej (po to sa wyjatki, zeby decydowac gdzie jest obslugujesz i zeby pare wartsw przeskakiwaly, bez zwracania bool jak w C)